### PR TITLE
Initial commit Docker

### PR DESCRIPTION
**Pull request by ayung92**

### CODE REVIEW

File: 4817_Sprint/8Aeron/receiver.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import socket
+import sys
+
+host = "localhost"
+#host = "172.20.0.1"
+
+port = 50030
+#s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM | socket.SO_REUSEADDR)
```
Comment (adiazQualogy) [inline 4817_Sprint/8Aeron/receiver.py]: remove commented code

File: 4817_Sprint/8Aeron/receiver.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+s = socket.socket(
+    socket.AF_INET, socket.SOCK_STREAM)
+s.bind((host, port))
+s.listen(2)
+print "waiting on port:", port
+while 1:
+    #data, addr = s.recvfrom(1024)
```
Comment (adiazQualogy) [inline 4817_Sprint/8Aeron/receiver.py]: remove commented code

File: 4817_Sprint/8Aeron/sender.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+sock.sendall(message)
+
+    # Look for the response
+amount_received = 0
+amount_expected = len(message)
+    
+while amount_received < amount_expected:
+    data = sock.recv(16)
+    amount_received += len(data)
+    #print >>sys.stderr, 'received "%s"' % data
```
Comment (adiazQualogy) [inline 4817_Sprint/8Aeron/sender.py]: remove commented code

File: 4817_Sprint/8Aeron/receiver.py (hunk 1 above)
Comment (ssoegriemsingh) [reply 4817_Sprint/8Aeron/receiver.py]: What he said ^

**Pull request merged**